situp-plugins/common/src/main/java/com/amazon/situp/plugins/source/FileSource.java

```diff
@@ -19,10 +21,13 @@
 
 @SitupPlugin(name = "file", type = PluginType.SOURCE)
 public class FileSource implements Source<Record<String>> {
+    private static final Logger LOG = LoggerFactory.getLogger(FileSource.class);
     private static final String ATTRIBUTE_PATH = "path";
+    private static final String ATTRIBUTE_TIMEOUT = "write_timeout";
     private final String filePathToRead;
+    private final int writeTimeout;
+    private final String pipelineName;
     private boolean isStopRequested;
-    private static final String SAMPLE_FILE_PATH = "src/resources/file-test-sample.txt";
     private static final int WRITE_TIMEOUT = 5_000;
```

Comment:
Can you group all the variables together by their classifications? E.g. PSF at top, PF, then P?